# Goal - Either a set of target joints OR a target point.
#        (If both are given, the joints will take precedent and the point will be ignored.)
float64[] target_joints         # [Right_tilt, Right_lift, Right_rotate, Right_elbow]
geometry_msgs/PointStamped target_point
---
# Result - Success result & final arm position
bool success
float64[] arm_position          # [Right_tilt, Right_lift, Right_rotate, Right_elbow]
---
# Feedback - Current arm position
float64[] arm_position          # [Right_tilt, Right_lift, Right_rotate, Right_elbow]
